python/kwiver/vital/types/types_module.cxx

```diff
@@ -1,5 +1,5 @@
 /*ckwg +29
- * Copyright 2019 by Kitware, Inc.
+ * Copyright 2019-2020 by Kitware, Inc.
```

Comment:
Curious about a file who's only change is the copyright date. Doesn't seem right.

Reply:
I'm not sure what's going on with the change on Github, but the `using namespace kwiver::vital::python` was added this year, as well as a few other things based on my local blame.

Reply:
Good enough for me.